modules/system/assets/ui/js/popover.js

```diff
@@ -41,6 +41,7 @@
     }
 
     Popover.prototype.hidePopover = function() {
+        $(window).off('resize.popoverReposition')
```

Comment:
@Teranode you'll need to remove this part as well. And you can unnamespace the listener.